README.md

```diff
@@ -150,26 +150,26 @@ income_general               income
  Account                       Balance             
 ---------------------------------------
 asset
-  checking                     1,300               
-  savings                      2,000               
+  checking                     1300.00               
+  savings                      2000.00               
 liability
-  credit_card                  -456                
+  credit_card                  -456.00                
```

Comment:
line 156 should sum to be `-456.50`, based on the transactions examples below it.